app/routes.tsx

```diff
@@ -1,31 +1,37 @@
-// @flow
+import * as express from "express"
+import * as React from "react"
+import * as ReactDOMServer from "react-dom/server"
+import * as Relay from "react-relay"
 
-import express from 'express'
-import type { $Request, $Response, NextFunction } from 'express' // eslint-disable-line no-duplicate-imports
-
-import React from 'react'
-import ReactDOMServer from 'react-dom/server'
-
-import Relay from 'react-relay' // eslint-disable-line no-unused-vars
-import IsomorphicRelay from 'isomorphic-relay'
+import IsomorphicRelay from "isomorphic-relay"
 
 import { artsyRelayMiddleware } from './relay/config'
 import { ArtistQueryConfig } from './relay/root_queries'
 
-import ReactNativeWebArtist from './containers/react-native-web/artist'
 import PureReactArtist from './containers/pure-react/artist'
 import ReactInlineCSSArtist from './containers/react-inline-css/artist'
+import ReactNativeWebArtist from './containers/react-native-web/artist'
 
 import { StyleSheetServer } from 'aphrodite'
 import ReactAphroditeArtist from './containers/react-aphrodite/artist'
 
-import { SheetsRegistryProvider, SheetsRegistry } from 'react-jss'
+import {SheetsRegistry, SheetsRegistryProvider } from 'react-jss'
 import ReactJSSArtist from './containers/react-jss/artist'
 
 const app = express.Router()
 
 app.use(artsyRelayMiddleware)
 
+app.get("/", (req: any, res: any, next: any) => {
+  res.send(`<html><body><ul>
+  <li><a href='/pure-react/artist/banksy'>pure-react</a></li>
+  <li><a href='/react-inline-css/artist/banksy'>react-inline-css</a></li>
+  <li><a href='/react-aphrodite/artist/banksy'>react-aphrodite</a></li>
+  <li><a href='/react-native-web/artist/banksy'>react-native-web</a></li>
+  <li><a href='/react-jss/artist/banksy'>react-jss</a></li>
+  `)
```

Comment:
Where. Are. The. Close. Tags???!!!

![Giphy](https://media2.giphy.com/media/KP7saBvJaNnc4/giphy.gif)